src/webapp/utils/analysis.tsx

```diff
@@ -2,16 +2,26 @@ import i18n from "$/utils/i18n";
 import { ProgressStatus } from "../components/progress-status/ProgressStatus";
 import { QualityAnalysis } from "$/domain/entities/QualityAnalysis";
 import { TableColumn } from "@eyeseetea/d2-ui-components";
+
 import styled from "styled-components";
+import { Tag } from "../components/tag/Tag";
 
-function mapAnalysisStatusToColor(sectionStatus: string) {
+export function mapAnalysisStatusToColor(sectionStatus: string) {
```

Comment:
I think we should use another function `mapIssueActionStatusToColor` since the analysis can have different values. It could be confusing if we mix all the possible values.

Reply:
done!